wave_utils/include/wave/utils/math.hpp

```diff
@@ -34,6 +34,7 @@ void vec2mat(std::vector<double> x, int rows, int cols, MatX &y);
 void mat2vec(MatX A, std::vector<double> &x);
 double wrapTo180(double euler_angle);
 double wrapTo360(double euler_angle);
+Mat3 skew_symmetric_matrix(Vec3 a);
 
```

Comment:
Just a naming thing but do you think if we can, we should borrow names from Matlab? e.g `skewdec` ([matlab doc](http://www.mathworks.com/help/robust/ref/skewdec.html)) for this instanace? I have already borrowed from `wrapTo180` and `wrapTo360`.